pkg/notify/notify.go

```diff
@@ -0,0 +1,30 @@
+package notify
+
+import (
+    "os"
+    "time"
+    "encoding/json"
+    "strconv"
+
+    "github.com/slack-go/slack"
+)
+
+const slackUsername = "a9s-aha-cli"
+
+func Notify(msg_text string) error {
```

Comment:
`notify.Notify` sound strange, but I don't know a better name (maybe ```notify.Send``` is better?).